sources/packages/api/src/route-controllers/application/application.student.controller.ts

```diff
@@ -10,6 +10,7 @@ import {
   Patch,
```

Comment:
For the client type specific controllers, we follow the convention of *.<clienttypebaseroute>.controller.ts 

Hence our base route is student**_s_** instead of student. File name should be application.students.controller.ts